src/Data/IntMap/Internal.purs

```diff
@@ -3,18 +3,24 @@ module Data.IntMap.Internal where
 
 import Data.Function (Fn2 (), runFn2)
 import Data.Int.Bits ((.^.), complement, (.&.), (.|.), zshr)
-import Prelude (class Ord, class Eq, (*), (-), otherwise, (==), (<), compare, eq)
+import Prelude
```

Comment:
Turns out: no. Purescript only warns when *multiple* unqualified imports are used. So using just Prelude here is OK.